index.html

```diff
@@ -50,6 +50,264 @@ <h1>I'm Matunda. Glad to see you!</h1>
+            <div class="see-project">
+              <a href="#"><p>See Project</p></a>
+            </div>
+          </div>
+
+        </div>
+        <!--Grid card 3-->
+        <div class="grid-card small-card">
+          <div class="snapshot">
+            <img src="./images/grd3.svg">
+          </div>
+          <div class="project-details">
+            <div class="project-name"><h2>Tonic</h2></div>
+          
+            <div class="project-more-details">
+              <div class="client">
+                <p>CANOPY</p>
+                              
+              </div>
+              <img class="counter-img" src="./images/Counter.svg"> 
+              <div class="role">
+                <p>Back End Dev</p> 
+                
+              </div>
+              <img class="counter-img" src="./images/Counter.svg"> 
+              <div class="year">
+                <p>2015</p>
+              </div>
+            </div>
+
+          </div>
+          <div class="summary">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          </div>
+          
+            <ul class="tags">
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>Javascript</li>
+            </ul>
+           
+          <div class="action">
+            <div class="see-project">
+              <a href="#"><p>See Project</p></a>
+            </div>
+          </div>
+
+        </div>
+        <!-- Grid card 4 -->
+
+        <div class="grid-card big-card">
+          <div class="snapshot">
+            <img src="./images/grd4.svg">
+          </div>
+          <div class="project-details">
+            <div class="project-name"><h2>Multi-Post<br>Stories         
+            </h2></div>
+          </div>
+            <div class="project-more-details">
+              <div class="client">
+                <p>CANOPY</p>
+                              
+              </div>
+              <img class="counter-img" src="./images/Counter.svg"> 
+              <div class="role">
+                <p>Back End Dev</p> 
+                
+              </div>
+              <img class="counter-img" src="./images/Counter.svg"> 
+              <div class="year">
+                <p>2015</p>
+              </div>
+            </div>
+
+          
+          <div class="summary">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          </div>
+          
+            <ul class="tags">
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>Javascript</li>
+            </ul>
+           
+          <div class="action">
+            <div class="see-project">
+              <a href="#"><p>See Project</p></a>
+            </div>
+          </div>
+
+        </div>
+
+
+      </div>
+    </section>
+    <!--About Me Section-->
+    <section>
+      <div class="about">
+        <div class="top">
+          <h1>About Me</h1>
```

Comment:
- [ ] Since this is not the main title of the webpage, It will be nice to use another header tag `hx` than the `h1`. You can use `(examples: <h1>,<h2>,<h3>, etc...)` for semantics reason. It also advises using only one `h1` tag per page.